client/src/app/containers/Game/types.ts

```diff
@@ -0,0 +1,17 @@
+import { User } from '../../../auth/slice';
+
+export interface ActionInputProps {
+    isBlocking: boolean;
+}
+
+export interface CreateGameResponse {
+    id: number;
+    players: User[];
+    player_colors: {
+        [key: string]: string;
+    };
+    blocking_players: {
+        [key: string]: string;
+    };
```

Comment:
:book: is there a good way to leave comments about what these maps are?
```ts
// player_colors is a map of playerID -> peg color
// blocking_players is a map of playerID -> blocking reason
```

Reply:
^ that looks like a good way to me. Will add.